enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -317,11 +285,57 @@ to WMCO 9.0.0, the proxy support will be available.
 The implementation history can be tracked by following the associated work items in Jira and source code improvements in
 the WMCO Github repo.
 
-Planned GA of the feature: WMCO v9.0.0 in OCP 4.14
-
 ## Alternatives & Justification
 
-* A workaround that would deliver the same value proposed by this enhancemnt would be to validate and provide guidance
+* A workaround that would deliver the same value proposed by this enhancement would be to validate and provide guidance
   to make cluster administrators responsible for manually propogating proxy settings to each of their Windows nodes, and
   underlying OpenShift managed components. This is not a feasible alternative as even manual node changes can be
   ephemeral. WMCO would reset config changes to OpenShift managed Windows services in the event of a node reconcilition.
+* There is another possible way for WMCO to [retrieve the proxy variables](#configuring-proxy-environment-variables). WMCO
+  can read the environment variables from its own pod spec (vars are injected + updated by OLM). The difficulty of this
+  approach comes from figuring out when we need to update node's env vars. Ideally such reconfiguring happens only when
+  the values change in the pod spec, but how can we detect if the proxy env vars changed or the operator just restarted 
+  for some other reason? We want to avoid kicking off reconfigurations of all nodes every time the operator restarts.
+* In order to avoid a system reboot after setting node environment variables, we can reconcile services by setting their
+  environment variables, and then restarting the services. These changes are ephemeral, though, if the instance reboots.
+  ```powershell
+  [string[]] $envVars = @("HTTP_PROXY=http://<username>:<pswd>@<ip>:<port>", "NO_PROXY=123.example.com,10.88.0.0/16")
+  Set-ItemProperty HKLM:SYSTEM\CurrentControlSet\Services\<$SERVICE_NAME> -Name Environment -Value $envVars
+  Restart-Service <$SERVICE_NAME>
+  ```
+* Also note that there is another way to get the [trusted CA data](#configuring-custom-trusted-certififactes) required
+  rather than accessing the ConfigMap directly, but I prefer not to follow it. It leaves open the same concern about
```

Comment:
I suggest saying `but it leaves open the same concern...` taking individual preference out of the statement and just stating why.